lms/static/js/spec/student_profile/learner_profile_view_spec.js

```diff
@@ -123,12 +123,7 @@ define(['backbone', 'jquery', 'underscore', 'common/js/spec_helpers/ajax_helpers
 
             beforeEach(function () {
                 loadFixtures('js/fixtures/student_profile/student_profile.html');
-                TemplateHelpers.installTemplate('templates/fields/field_readonly');
-                TemplateHelpers.installTemplate('templates/fields/field_dropdown');
-                TemplateHelpers.installTemplate('templates/fields/field_textarea');
-                TemplateHelpers.installTemplate('templates/fields/field_image');
                 TemplateHelpers.installTemplate('templates/fields/message_banner');
```

Comment:
It broke the tests when I took it out. I'll take a quick pass through and work out why.